src/main/java/ch/njol/skript/conditions/CondIsGlowing.java

```diff
@@ -0,0 +1,51 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *
+ * Copyright 2011-2017 Peter Güttinger and contributors
+ */
+package ch.njol.skript.conditions;
+
+import org.bukkit.entity.Entity;
+
+import ch.njol.skript.conditions.base.PropertyCondition;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+
+@Name("Is Glowing")
+@Description("Checks whether an entity is glowing.")
+@Examples({"player is glowing:",
+		"	message \"You have been blessed!\""})
```

Comment:
I believe Snow told me in the past, we should be using "\t" rather than spaces "    " for examples

Reply:
Sorry, correct me if I'm wrong, but
``Tabs, no spaces (unless in code imported from other projects) ** No tabs/spaces in empty lines`` (if this is the line you're referring to) is referring to the space character, but I'm using a proper tab character in the example, albeit not in \t form, which should still be valid?